todo_app/templates/index.html

```diff
@@ -12,7 +12,25 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        {% for todo in todos %}
+          <ul>  {{todo.title}} - {{todo.status}} <a class="btn btn-primary" href="/modify/{{ todo.id }}" role="button">Modify</a></ul>
+            <!-- I didn't like the way they looked, so commenting them out
+              <form action="{{ url_for('update_todo')}}" method="post"><input type="hidden" name="id" value="{{ todo.id }}"> <input type="hidden" name="status" value="In progress"><input type="submit" value="In Progress" class="btn btn-info" > </form> 
+            <form action="{{ url_for('update_todo')}}" method="post"><input type="hidden" name="id" value="{{ todo.id }}"> <input type="hidden" name="status" value="Completed"> <input type="submit" value="Completed" class="btn btn-success" > </form>  
+            <form action="{{ url_for('remove_todo')}}" method="post"><input type="hidden" name="id" value="{{ todo.id }}"> <input type="submit" value="Remove" class="btn btn-danger"> </form> 
+            -->
```

Comment:
That's fine - if you prefer having them within the modify page then not an issue, but if you do stick with that then let's remove this code here; we can always recover it with Git later if we want to, but leaving it leaves the potential to confuse people/create extra work (e.g. should this be maintained if the endpoints change?)